vnext/Desktop/React.Windows.Desktop.vcxproj

```diff
@@ -129,10 +131,16 @@
       <DependentUpon>ABI\NativeTracing.idl</DependentUpon>
       <ObjectFileName>$(IntDir)\ABI\</ObjectFileName>
     </ClCompile>
-    <ClCompile Include="ABI\ReactNativeHost.cpp">
+    <ClCompile Include="..\Microsoft.ReactNative\ReactNativeHost.cpp">
       <DependentUpon>..\Microsoft.ReactNative\ReactNativeHost.idl</DependentUpon>
       <ObjectFileName>$(IntDir)\ABI\</ObjectFileName>
     </ClCompile>
+    <ClCompile Include="..\Microsoft.ReactNative\IReactPropertyBag.cpp" />
```

Comment:
>..\Microsoft.ReactNative [](start = 24, length = 24)

Nit: Should we declare a property with the folder (in case projects move, this would be easier to updaet. I fully recnogize the pattern is used above already :)

Reply:
Possibly. In the long run, we'd like to unify the two projects, creating a single "core" DLL.

---
In reply to: [426966866](https://github.com/microsoft/react-native-windows/pull/4924#discussion_r426966866) [](ancestors = 426966866)